components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -41,16 +44,37 @@ const HighlightInputForm = (): JSX.Element => {
   };
 
   // Handle submit highlights
-  const handlePostHighlight = (e: React.FormEvent<HTMLFormElement>) => {
+  const handlePostHighlight = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    setIsDivFocused(false);
+
     // Trigger api call to post highlight
-    setBodyText("");
-    setTitle("");
+    const pullLink = bodyText.match(/\bhttps?:\/\/\S+/gi);
+    const [url] = pullLink || [];
+    const highlight = bodyText.replace(url as string, "");
+
+    if (url === null || undefined || "" || []) {
```

Comment:
This will always show an error, the variable must be in each block

```suggestion
    if (url === null || url === []) {
```